AVRO_TENSOR_DATASET.md

```diff
@@ -0,0 +1,354 @@
+# AvroTensorDataset
+
+AvroTensorDataset is a `tf.data.Dataset` implementation. It consumes records from one or more Avro files. The supported schema are discussed with more details in later section.
+
+AvroTensorDataset loads Avro records from files into a dict of tensors.
+The output dict has feature name as key and tf.Tensor or tf.SparseTensor
+as value. The output tensor values are batched with the user defined
+batch size. 
+
+## Python API
+
+The AvroTensorDataset supports the same features as AvroRecordDataset. A minimal example is given below:
```

Comment:
Perhaps we can delete `The AvroTensorDataset supports the same features as AvroRecordDataset.` ? Since user may not know what is `AvroRecordDataset`. Also AvroTensorDataset does not support all schemas that AvroRecordDataset does.

Reply:
makes sense, updated.